eng/build.sh

```diff
@@ -444,6 +445,11 @@ while [[ $# > 0 ]]; do
       fi
       ;;
 
+      -pgoinstrument)
+      arguments="$arguments /p:pgoinstrument=true"
```

Comment:
Nit: better use consistent casing for this MSBuild property.
```suggestion
      arguments="$arguments /p:PgoInstrument=true"
```